cmd/fluxctl/install_cmd.go

```diff
@@ -52,6 +52,7 @@ fluxctl install --git-url 'git@github.com:<your username>/flux-get-started' --gi
 		"do not scan container image registries to fill in the registry cache")
 	cmd.Flags().StringVarP(&opts.outputDir, "output-dir", "o", "", "a directory in which to write individual manifests, rather than printing to stdout")
 	cmd.Flags().BoolVar(&opts.AddSecurityContext, "add-security-context", true, "Ensure security context information is added to the pod specs. Defaults to 'true'")
+	cmd.Flags().StringVar(&opts.CacheBackend, "cache-backend", "memcached", "Select the docker images repository cache backend. Defaults to 'memcached'")
```

Comment:
Despite the fact that other random flags have found their way in here (I'm looking at you, `--add-security-context`), I don't think it's necessary for the cache backend to be an option here, especially not while it's experimental.

Reply:
from an end-user perspective, if I want to give a try to a different cache service, I would really enjoy this flag: furthermore: at first glance, it looks like a Helm alternative and I get your point, probably `flux install` should just provide a battle-tested installation without so many parameters.

If we want to remove the `cache-backend` flag, I guess the same fate has to be reserved for `add-security-context`.